.travis.yml

```diff
@@ -15,6 +15,7 @@ notifications:
 templates:
```

Comment:
is this template part really necessary?

Reply:
It simplify syntax, so I recommend keep it.